modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/web/internal/portlet/action/test/ObjectDefinitionExportImportTest.java

```diff
@@ -72,6 +75,112 @@ public void testExportImportObjectDefinition() throws Exception {
 			_baseObjectDefinitionJSONString,
 			_baseObjectDefinitionJSONString, null,
 			"TestObjectDefinitionJSON");
+
+		JSONObject actualJSONObject = JSONFactoryUtil.createJSONObject(
+			_baseObjectDefinitionJSONString
+		).put(
+			"externalReferenceCode", "TESTOBJECTDEFINITIONGERMAN"
+		).put(
+			"name", "TestObjectDefinitionInGerman"
+		);
+
+		JSONObjectDefinitionTestUtil.addLocalizedLabel(
```

Comment:
For your test case, shouldnt this be a setLocalizedLabel? Since it should only have the german label and not the default english one?